test/lib_i2s/i2s_slave_test/src/main.c

```diff
@@ -23,17 +23,16 @@ xclock_t bclk = XS1_CLKBLK_1;
 #define I2S_LOOPBACK_LATENCY 1
 
 #if SMOKE == 1
-#define NUM_BCLKS 1
-#define NUM_BCLKS_TO_CHECK 1
-static const unsigned bclk_freq_lut[NUM_BCLKS] = {
-  1228800
+#define NUM_LRCLKS 1
+#define NUM_LRCLKS_TO_CHECK 1
+static const unsigned lr_freq_lut[NUM_LRCLKS] = {
+  192000
 };
 #else
-#define NUM_BCLKS 12
-#define NUM_BCLKS_TO_CHECK 3
-static const unsigned bclk_freq_lut[NUM_BCLKS] = {
-  1228800, 614400, 384000, 192000, 44100,
-  22050, 96000, 176400, 88200, 48000, 24000, 352800
+#define NUM_LRCLKS 12
+#define NUM_LRCLKS_TO_CHECK 3
```

Comment:
Approved, but, why are these 12 and 3? To be frank we could just make them 6 and 6, no point having clocks if we're not going to use them